src/linear_algebra/projective_space/subspace.lean

```diff
@@ -127,6 +121,26 @@ instance : complete_lattice (subspace K V) :=
 instance subspace_inhabited : inhabited (subspace K V) :=
 { default := ⊤ }
 
+/-- The span of a set of points is contained in a subspace if and only if the set of points is
+contained in the subspace. -/
+lemma span_le_subspace_iff {S : set (ℙ K V)} {W : subspace K V} : span S ≤ W ↔ S ⊆ W :=
+gi.gc S W
+
+/-- If a set of points is a subset of another set of points, then its span will be contained in the
+span of that set. -/
+lemma span_mono {S T : set (ℙ K V)} (h : S ⊆ T) : span S ≤ span T :=
+galois_connection.monotone_l gi.gc h
+
+/-- The supremum of two subspaces is equal to the span of their union. -/
+lemma sup_eq_span_union (W S : subspace K V) : W ⊔ S = span (W ∪ S) :=
+by { apply symm, exact (galois_insertion.l_sup_u gi W S) }
```

Comment:
I think a better lemma is:
```suggestion
lemma span_union (S T : set (ℙ K V)) : span (S ∪ T) = span S ⊔ span T := (@gi K V _ _ _).gc.l_sup
```

Reply:
You might also want to look at the API for https://leanprover-community.github.io/mathlib_docs/linear_algebra/span.html#submodule.span to see what else could be added.